packages/backend/src/tokens/interstitial.ts

```diff
@@ -93,7 +93,7 @@ export function loadInterstitialFromLocal(
                 ${domain ? `script.setAttribute('data-clerk-domain', '${domain}');` : ''}
                 ${proxyUrl ? `script.setAttribute('data-clerk-proxy-url', '${proxyUrl}')` : ''};
                 script.async = true;
-                script.src = '${getScriptUrl(domain || frontendApi, pkgVersion)}';
+                script.src = '${getScriptUrl(frontendApi, pkgVersion)}';
```

Comment:
@panteliselef wonder why the domain was causing issues?

Reply:
It's better to ask the primary domain's fapi to serve clerkJS. This is not a real fapi request and is better routed to primary.

Reply:
and interstitial from BAPI already follows the logic @gkats described
This change ensures that both interstitial scripts have the same behaviour